phoonnx/model_manager.py

```
@@ -61,6 +61,11 @@ def __post_init__(self):
             self.config = self.config or VoiceConfig.from_dict(config)
             self.config.lang_code = self.lang  # sometimes the config is wrong
 
+        try:
+            self.config.lang_code = self.lang = standardize_tag(self.config.lang_code)
+        except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)